gherkin/php/src/TokenMatcher.php

```diff
@@ -6,13 +6,16 @@
 
 use Cucumber\Gherkin\Parser\TokenType;
 use Cucumber\Gherkin\Parser\TokenMatcherInterface;
+use Cucumber\Messages\Step\KeywordType;
 
 final class TokenMatcher implements TokenMatcherInterface
 {
     private const LANGUAGE_PATTERN = "/^\\s*#\\s*language\\s*:\\s*([a-zA-Z\\-_]+)\\s*$/";
     private GherkinDialect $currentDialect;
     private int $indentToRemove = 0;
     private ?string $activeDocStringSeparator = null;
+    /** @var array<array-key, array<array-key, KeywordType>> */
```

Comment:
We should be specific here

I believe it's an `array<string, list<KeywordTypes>`?